setup.py

```diff
@@ -8,6 +10,18 @@
     filepath = "optimum/intel/version.py"
     with open(filepath) as version_file:
         (__version__,) = re.findall('__version__ = "(.*)"', version_file.read())
+    if __version__.endswith(".dev0"):
+        dev_version_id = "unknown_version"
```

Comment:
would prefer to not update `__version__` if we're not able to get the latest commit

Reply:
updated